modules/transactions/src/main/java/org/apache/ignite/internal/tx/TxManager.java

```diff
@@ -224,6 +224,14 @@ CompletableFuture<Void> cleanup(
      */
     CompletableFuture<Void> vacuum();
 
+    /**
+     * Kills a local transaction by their id. The behavior is similar with the transaction rollback.
+     *
+     * @param txId Transaction id.
+     * @return Future will be completed with value true if the transaction was started locally and completed.
```

Comment:
```suggestion
     * @return Future will be completed with value true if the transaction was started locally and completed completed by this call.
```
What happens if the transaction has already been completed by another call but has not yet been `vacuumed`?